python/tidy/servo_tidy/tidy.py

```diff
@@ -74,6 +74,10 @@
     " accessible to\n// web pages."
 ]
 
+DIRECTORIES_FOR_FILETYPES = {
+    "components/script/dom/webidls": [".webidl"]
+}
+
```

Comment:
Like @Ms2ger said, this should live in the config file (`servo-tidy.toml` in the root dir), and the dict should be obtained from the `config` variable.